rpcs3/Emu/RSX/CgBinaryFragmentProgram.cpp

```diff
@@ -47,18 +47,16 @@ void CgBinaryDisasm::AddCodeAsm(const std::string& code)
 std::string CgBinaryDisasm::GetMask()
 {
 	std::string ret;
+	ret.reserve(4);
 
-	static const char dst_mask[4] =
-	{
-		'x', 'y', 'z', 'w'
-	};
+	static const std::string dst_mask = "xyzw";
```

Comment:
```suggestion
	static constexpr std::string_view dst_mask = "xyzw";
```

Reply:
And there is a question, it's equal everywhere. Maybe these should be moved out of method scope.